src/core/best-practices.md

```diff
@@ -54,9 +54,61 @@ We recommend starting to record prior to calibration. This is valuable because i
 
 ## Participants
 
-The less movement you participants will exhibit during an experiment, the more accurate your results will be. This includes movements of head, but also of facial muscles. In order to minimize unnecessary movement, you should:
+The less movement your participants will exhibit during an experiment, the more accurate your results will be. This includes movements of head, but also of facial muscles. In order to minimize unnecessary movement, you should:
 
 - Make sure the participant is in a comfortable position that they can maintain during the experiment.
 - Make sure the participant can comfortably read all instructions and does not need to squint.
 - Note: If the participant can keep a relaxed face during the experiment this will minimize slippage.
 - Ask your participant to not touch the headset (e.g. by scratching their head). If they need to interrupt the block, they should ask and you will need to restart the block and recalibrate.
+
+## Fixation Filter Thresholds
+
```

Comment:
Maybe we should link the existing fixation detector docs here for reference

Reply:
Agreed